### PR TITLE
Update Bitcanna Testnet chain.json

### PR DESCRIPTION
**Pull request by kjincgroup**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Any chance we can also leave the old version in the version array, but then also adding the new object?

**Comment by kjincgroup** [comment]
Thank you for your suggestion, please double check.

**Comment by JeremyParish69** [comment]
> Thank you for your suggestion, please double check.

Thanks, but I mean having both v1.6 and v2.0 within the versions array.

File: testnets/bitcannadevnet/chain.json [hunk 1]
```diff
@@ -22,25 +22,25 @@
...
+        "recommended_version": "v1.6.3",
         "compatible_versions": [
-          "v1.6.2"
+          "v1.6.3"
         ],
         "binaries": {
-          "linux/amd64": "https://github.com/BitCannaGlobal/bcna/releases/download/v1.6.2/bcna_linux_amd64.tar.gz"
+          "linux/amd64": "https://github.com/BitCannaGlobal/bcna/releases/download/v1.6.3/bcna_linux_amd64.tar.gz"
         }
       }
```
Comment (JeremyParish69) [inline testnets/bitcannadevnet/chain.json:45]: adding this here in the versions array:
  ```
},
{
    "name": "v2.0",
    "recommended_version": "v2.0.1-rc6",
    "compatible_versions": [
      "v2.0.1-rc6"
    ],
    "binaries": {
      "linux/amd64": "https://github.com/BitCannaGlobal/bcna/releases/download/v2.0.1-rc6/bcna_linux_amd64.tar.gz"
    }
  }
```

Comment (kjincgroup) [reply testnets/bitcannadevnet/chain.json:55]: Done.

**Pull request merged**